msteams-platform/concepts/deploy-and-publish/apps-publish-overview.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Overview of publish your app
+description: This is an overview of how to package and publish your app.
+ms.topic: conceptual 
+keywords: teams apps Upload
+---
+
+# Publish your apps
+Apps that you develop need to be published to your organization for the users to use them. There is a complete process to publish your apps that includes, selecting methods of distribution, packaging and uploading, and finally publishing.
```

Comment:
Maybe revise to cover all options for publishing. Publishing to an organization is only one of three.